Update aggregation-functions.md

amiratupsolver opened the pull request:
updated COUNT_IF using azure formatting style and Shy's instructions

yoniiny commented on getting-started/glossary/language-guide/functions/aggregation-functions.md:
```diff
@@ -39,6 +39,54 @@ Using this aggregation with primary key `id` and `COUNT(data)` produces the foll
+SET bool_expr_var = data = 'sample data';
+SET bool_expr_complex_var = data = 'sample data' AND extendeddata='apple';
+SELECT type,
+COUNT_IF(bool_expr_var) as data,
```
I think rather inline the SET statements, it'll be easier for people to read that way. I find that people don't really understand boolean expressions in set statements (maybe the syntax is confusing, since there are two = signs one after the other).

shanielh commented on getting-started/glossary/language-guide/functions/aggregation-functions.md:
````diff
@@ -39,6 +39,54 @@ Using this aggregation with primary key `id` and `COUNT(data)` produces the foll
+
+_Query:_
+```sql
+SET bool_expr_var = data = 'sample data';
+SET bool_expr_complex_var = data = 'sample data' AND extendeddata='apple';
+SELECT type,
+COUNT_IF(bool_expr_var) as data,
+COUNT_IF(bool_expr_complex_var) as apples
+  FROM stream
+      GROUP BY  type
````
```suggestion
SELECT type,
COUNT_IF(data = 'sample data') as data,
COUNT_IF(data = 'sample data' AND extendeddata='apple') as apples
    FROM stream
GROUP BY  type
```

amiratupsolver commented:
Removed:
               SET bool_expr_var = data = 'sample data';
               SET bool_expr_complex_var = data = 'sample data' AND extendeddata='apple';   

Changed:
               COUNT_IF(bool_expr_complex_var) as apples
To:               
               COUNT_IF(data = 'sample data' AND extendeddata='apple') as apples,
